File: Platform/codesignal/Intro/35.firstDigit/solution.py
import re

def solution(s):
  for x in s:
    if x.isdigit():
      return x
    
def solution2(s):
  return re.search('\d', s).group()

def solution3(s):
  for x in s:
    if '0' <= x <= '9':
      return x


print(solution("var_1__Int")) #1

File: Platform/codesignal/Intro/35.firstDigit/solution.js
// function solution(s) {
//     let arr = s.split('');
//     for (let i = 0; i < arr.length; i++) {
//         if (arr[i].charCodeAt() >= 48 && arr[i].charCodeAt() <= 57) {
//             return arr[i];
//         }
//     }
// }

// function solution2(s) {
//     return s.match(/\d/)[0]
// }

// function solution3(s) {
//     return s.match(/[0-9]/)[0]
// }

function solution4(s) {
  for (let i = 0; i < s.length; i++) {
    if (s[i] >= "0" && s[i] <= "9") {
      return s[i];
    }
  }
}

console.log(solution("var_1__Int"));